.editorconfig

```diff
@@ -2,4 +2,7 @@ root = true
 
 [*.cs]
 # Sort using directives with System.* appearing first
-dotnet_sort_system_directives_first = true
\ No newline at end of file
+dotnet_sort_system_directives_first = true
+
+# MSML_GeneralName: This name should be PascalCased
+dotnet_diagnostic.MSML_GeneralName.severity = none
```

Comment:
This shouldn't be checked in. We have this analyzer written so people don't violate this rule.

Reply:
removed

## Resolve